optimum/onnxruntime/base.py

```diff
@@ -178,21 +179,28 @@ def __init__(
                 self.value_sequence_length_idx = -1
 
     def prepare_inputs_for_merged(
-        self, input_ids: Optional[torch.LongTensor] = None, past_key_values: Optional[List[torch.FloatTensor]] = None
+        self,
+        input_ids: Union[None, torch.LongTensor, np.ndarray] = None,
+        past_key_values: Optional[Union[Tuple[torch.FloatTensor], Tuple[np.ndarray]]] = None,
```

Comment:
IMO make a choice here `Union[None, ...]` then `Optional`.

Reply:
Do you mean `Optional[Union[None, Tuple[torch.FloatTensor], Tuple[np.ndarray]]]`? What's the difference?

Reply:
Oh nvm got it, you were referencing the two type annotations!